constants/alums.ts

```diff
@@ -1,12 +1,13 @@
-enum Gender{
+enum Gender {
     Male = 'male',
     Female = 'female',
 }
 
 export const Alums = [
     {
         name: 'Richa',
-        designation: 'Software Developer, JP Morgan & Chase',
+        designation_a: 'Software Developer, JP Morgan & Chase',
+        designation_b: '',
```

Comment:
what's the purpose of having a second designation for alums? We can remove it ig. then handle on the card component using a v-if